js/deribit.js

```diff
@@ -168,25 +181,29 @@ module.exports = class deribit extends Exchange {
                         'buy',
                         'sell',
                         'edit',
+                        'edit_by_label',
                         'cancel',
                         'cancel_all',
                         'cancel_all_by_currency',
                         'cancel_all_by_instrument',
                         'cancel_by_label',
                         'close_position',
                         'get_margins',
+                        'get_mmp_config',
                         'get_open_orders_by_currency',
                         'get_open_orders_by_instrument',
                         'get_order_history_by_currency',
                         'get_order_history_by_instrument',
                         'get_order_margin_by_ids',
                         'get_order_state',
-                        'get_stop_order_history',
+                        'get_trigger_order_history',
```

Comment:
The name for this endpoint changed, I can't find the old name `get_stop_order_history` anywhere in the API, do we keep both or just get the new one?

Reply:
@samgermain looks like they renamed it, so we should indeed replace the old one with the new one:

<img width="871" alt="Screenshot 2021-11-24 at 05 45 28" src="https://user-images.githubusercontent.com/1294454/143162237-42f73d95-40a7-428d-814f-0d508acfd1f5.png">

Reply:
Although, we could keep both for backward-compatibility, since they say the old endpoint will also work: https://www.google.com/search?q=deribit+get_stop_order_history

Reply:
I added a deprecated comment beside it, I'll have to open another PR to add that in I guess